src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,27 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
+        List<Manufacturer> listOfManufacturers =
+                List.of(new Manufacturer("Toyota", "Japan"),
+                        new Manufacturer("Ford", "USA"),
+                        new Manufacturer("Accent", "South Korea"));
+        ManufacturerDao manufacturerDao
+                = (ManufacturerDao) injector.getInstance(ManufacturerDao.class);
 
+        for (Manufacturer manufacturer: listOfManufacturers) {
+            manufacturerDao.create(manufacturer);
+        }
+        System.out.println(manufacturerDao.get(1L));
```

Comment:
It is a bad practice to use magical IDs
```suggestion
        System.out.println(manufacturerDao.get(manufacturer.getId()));
```